src/gui/application.cpp

```diff
@@ -250,6 +250,12 @@ Application::Application(int &argc, char **argv)
         }
     }
 
+    if (_theme->doNotUseProxy()) {
+        ConfigFile().setProxyType(QNetworkProxy::NoProxy);
+    }
+
+    ConfigFile cfgFile;
```

Comment:
why is this line needed ?

Reply:
you seem not to use `configFile` variable

Reply:
@mgallien thx, and, fixed